src/types/react-fix.d.ts

```
@@ -0,0 +1,27 @@
+declare module 'scheduler/tracing' {
+  export interface Interaction {
+    readonly id: number
+    readonly name: string
+    readonly timestamp: number
+  }
+
+  export interface InteractionsRef {
+    current: Set<Interaction>
+  }
+
+  export function unstable_clear(callback: Function): any
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)